### DISCUSSION TITLE
Need help recovering switches

### DISCUSSION THREAD

**Discussion by mcarson75** [arendst/Tasmota#20248] [Q&A]
I have been running my devices on 12.4.0 for a long time.  I saw that 13.3.0 is out and decided it was time to upgrade.  I upgraded my first switch (Treatlife SS-01)  OTA with no issues - everything works fine.  I then proceeded to upgrade a few more.  A Shelly 1 also upgraded fine with no issues.  I then proceeded to do OTA on two more switches, a second Treatlife SS-01 and a Treatlife DS-01.

Neither of them has come back up.  I can see the onboard SSID for each of them, but if I connect to the SSID and try to go to 192.168.4.1 I receive `ERR_EMPTY_RESPONSE` and cannot connect.  In addition, the SS-01 does not work as a switch either - hitting the button doesn't do anything.  The DS-01 still works.

Short of soldering pins and reflashing that way, is there any other way to recover these?  I don't know what has happened or why one SS-01 seems to be fine and the other is not.

**Comment by sfromis** [comment]
If by "onboard SSID" you mean a name of ESP-something, that is normally a sign of it being bricked, wired flashing needed. Have no guess about what happened.

**Reply by mcarson75** [reply]
The onboard SSID that I was seeing was `tasmota-garage-light` which is the hostname in the Tasmota config.  Not the one based on MAC address you would see on a fresh flash.  Regardless, I soldered/re-flashed the SS-01.  Still working on the DS-01.

**Reply by mcarson75** [reply]
@sfromis Any other ideas about how to recover?  Most of my other devices upgraded fine OTA but I did have one more DS-01 do the same thing.  It is still working, so I think Tasmota is running - I just can't connect to it, and it doesn't connect to Wifi.  As mentioned, it is broadcasting the SSID associated with the configured hostname.  I can attach to the SSI and get an IP address, but when I to 192.168.4.1 I get `ERR_EMPTY_RESPONSE`.  Rebooting does not fix.

**Reply by sfromis** [reply]
Well, as I can't see anything about device status, I have no clear way forward outside of wired flashing. It might be useful to check what it says on serial, but when you already have wired connection, easier to just flash it....

**Reply by mcarson75** [reply]
Yeah I understand.  I'm not able to connect to it serially to flash either - it says there is no serial connection.  If I remember correctly, to flash these DS-01s you have to use an external power supply with a slightly higher voltage.  Been a while since I have done it though and my external power supply failed, so I can't test it at the moment.

My plan right now is to just replace the ESP8266 with an ESP32.
